templates/graphdb/statefulset.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     {{- tpl ( toYaml . ) $ | nindent 4 }}
   {{- end }}
 spec:
-  replicas: {{ $.Values.replicas }}
+  replicas: {{ .Values.replicas }}
   serviceName: {{ include "graphdb.fullname.service.headless" . }}
   updateStrategy:
     type: RollingUpdate
```

Comment:
We can think about making this a config. I think `OnDelete` is better in our case.

Reply:
Already on my TODO list